app/digidoo/hospitals/create/template.hbs

```diff
@@ -0,0 +1,55 @@
+<h2>Create</h2>
+<form action="/hospitalsmvc/create" method="post">
+    <div class="form-horizontal">
+        <h1>Hospital Information</h1>
+        <hr/>
+        <div class="form-group">
+            <label class="control-label col-md-2" for="Name">Name</label>
+            <div class="col-md-10">
+                <input class="form-control text-box single-line" id="Name" name="Name" type="text" value="" />
```

Comment:
These inputs are still regular inputs and Ember will not have any idea that the user is changing them. Look at the Input helpers from the Ember Docs: http://guides.emberjs.com/v2.0.0/templates/input-helpers/